ERCS/erc-chain_specific_addresses.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Chain-specific addresses using ENS
+description: A unified chain-specific address format that allows specifying the account as well as the chain on which that account intends to transact. 
+author: Sam Kaufman (@SampkaML), Marco Stronati (@paracetamolo), Yuliya Alexiev (@yuliyaalexiev), Jeff Lau (@jefflau), Sam Wilson (@samwilsn), Vitalik Buterin (@vbuterin)
+discussions-to: <URL>
```

Comment:
Please create a discussions topic in Eth Magicians.  
https://ethereum-magicians.org/c/ercs/57

It just needs to be a link to this PR.